### PR TITLE
PMM-7412 DBaaS: Add API for getting logs

### PR DESCRIPTION
**Pull request by jprukner**
[PMM-7412](https://jira.percona.com/browse/PMM-7412)

- [x] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1466

- [x] Links to other linked pull requests (optional).
  - https://github.com/percona/pmm/pull/675
  - https://github.com/percona-platform/dbaas-controller/pull/141
  - https://github.com/percona-platform/dbaas-api/pull/89
---
- [ ] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

File: services/management/dbaas/logs_service.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	controllerClient dbaasClient
+}
+
+// NewLogsService creates new LogsService.
+func NewLogsService(db *reform.DB, client dbaasClient) dbaasv1beta1.LogsAPIServer {
+	l := logrus.WithField("component", "logs_api")
+	return &LogsService{db: db, l: l, controllerClient: client}
+}
+
+func (s LogsService) GetLogs(ctx context.Context, in *dbaasv1beta1.GetLogsRequest) (*dbaasv1beta1.GetLogsResponse, error) {
```
Comment (jprukner) [inline services/management/dbaas/logs_service.go:43]: comment

**Comment by jprukner** [comment]
Rebased, solved conflicts.

**Comment by JiriCtvrtka** [comment]
@jprukner Test failed, please take a look.

**Review by JiriCtvrtka — APPROVED** [review]

**Review by askomorokhov — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Review by askomorokhov — APPROVED** [review]

**Comment by AlekSi** [comment]
@jprukner please resolve merge conflicts

**Review by percona-csalguero — APPROVED** [review]

**Pull request merged**